.storybook/preview.js

```diff
@@ -29,78 +28,47 @@ import '../assets/sass/wpdashboard.scss';
 import '../assets/sass/adminbar.scss';
 import '../assets/sass/admin.scss';
 import './assets/sass/wp-admin.scss';
+// Ensure all globals are setup before any other imports are run.
+import './polyfill-globals';
+import { resetGlobals } from './utils/resetGlobals';
 import { bootstrapFetchMocks } from './fetch-mocks';
-// TODO: Remove when legacy data API is removed.
-import { googlesitekit as dashboardData } from '../.storybook/data/wp-admin-admin.php-page=googlesitekit-dashboard-googlesitekit';
+import { WithTestRegistry } from '../tests/js/utils';
+import { enabledFeatures } from '../assets/js/features';
 
 bootstrapFetchMocks();
 
-const resetGlobals = () => {
-	global._googlesitekitLegacyData = cloneDeep( dashboardData );
-	global._googlesitekitLegacyData.admin.assetsRoot = '';
-	global._googlesitekitLegacyData.isStorybook = true;
-	global._googlesitekitBaseData = {
-		homeURL: 'http://example.com/',
-		referenceSiteURL: 'http://example.com/',
-		userIDHash: 'storybook',
-		adminURL: 'http://example.com/wp-admin/',
-		assetsURL: 'http://example.com/wp-content/plugins/google-site-kit/dist/assets/',
-		blogPrefix: 'wp_',
-		ampMode: false,
-		isNetworkMode: false,
-		isFirstAdmin: true,
-		isOwner: true,
-		splashURL: 'http://example.com/wp-admin/admin.php?page=googlesitekit-splash',
-		proxySetupURL: 'http://example.com/wp-admin/index.php?action=googlesitekit_proxy_setup&nonce=abc123',
-		proxyPermissionsURL: 'http://example.com/wp-admin/index.php?action=googlesitekit_proxy_permissions&nonce=abc123',
-		trackingEnabled: false,
-		trackingID: 'UA-000000000-1',
-	};
-	global._googlesitekitEntityData = {
-		currentEntityURL: null,
-		currentEntityType: null,
-		currentEntityTitle: null,
-		currentEntityID: null,
-	};
-	global._googlesitekitUserData = {
-		user: {
-			id: 1,
-			name: 'Wapuu WordPress',
-			email: 'wapuu.wordpress@gmail.com',
-			picture: 'https://wapu.us/wp-content/uploads/2017/11/WapuuFinal-100x138.png',
-		},
-		connectURL: 'http://example.com/wp-admin/index.php?action=googlesitekit_connect&nonce=abc123',
-		initialVersion: '',
-		verified: true,
-		userInputState: 'completed',
-		permissions: {
-			googlesitekit_authenticate: true,
-			googlesitekit_setup: true,
-			googlesitekit_view_posts_insights: true,
-			googlesitekit_view_dashboard: true,
-			googlesitekit_view_module_details: true,
-			googlesitekit_manage_options: true,
-			googlesitekit_publish_posts: true,
-		},
-	};
-};
-resetGlobals();
+// Decorators run from last added to first.
```

Comment:
This one just took my brain an extra second to parse; let's clarify so it makes sense immediately to as many folks as possible—some will prefer the original explanation so no harm in leaving both:

```suggestion
// Decorators run from last added to first. (Eg. In reverse order as listed.)
```